sdk/src/test/java/io/customer/sdk/CustomerIOClientTest.kt

```diff
@@ -0,0 +1,198 @@
+package io.customer.sdk
+
+import io.customer.base.data.ErrorResult
+import io.customer.base.error.ErrorDetail
+import io.customer.base.error.StatusCode
+import io.customer.base.utils.ActionUtils
+import io.customer.sdk.data.request.MetricEvent
+import io.customer.sdk.repository.IdentityRepository
+import io.customer.sdk.repository.PreferenceRepository
+import io.customer.sdk.repository.PushNotificationRepository
+import io.customer.sdk.repository.TrackingRepository
+import io.customer.sdk.utils.verifyError
+import io.customer.sdk.utils.verifySuccess
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mockito
+import org.mockito.Mockito.times
+import org.mockito.Mockito.verify
+import org.mockito.kotlin.any
+import org.mockito.kotlin.mock
+
+internal class CustomerIOClientTest {
+
+    private val preferenceRepository: PreferenceRepository = mock()
+    private val identityRepository: IdentityRepository = mock()
+    private val trackingRepository: TrackingRepository = mock()
+    private val pushNotificationRepository: PushNotificationRepository = mock()
+
+    private lateinit var customerIOClient: CustomerIOClient
+
+    @Before
+    fun setup() {
+        customerIOClient = CustomerIOClient(
+            identityRepository = identityRepository,
+            trackingRepository = trackingRepository,
+            pushNotificationRepository = pushNotificationRepository,
+            preferenceRepository = preferenceRepository
+        )
+    }
+
+    @Test
+    fun `verify client sends error when identify repo fails identifying`() {
+        Mockito.`when`(identityRepository.identify(any(), any()))
```

Comment:
I personally prefer when writing Android tests with mockito to use a static import for `when()`. 

```kt
Mockito.`when`()

to

when()
```

Do you have a preference? It's boilerplate to me to continuously see `Mockito.XXX` when writing tests.